_shared/project/conf/supervisord.conf

```diff
@@ -3,29 +3,32 @@ nodaemon=true
 environment=PYTHONUNBUFFERED="1"
 logfile=/dev/null
 logfile_maxbytes=0
-{% if cookiecutter.get("_directory") == "pyapp" %}
 
-[program:app]
-command=python3 -m {{ cookiecutter.package_name }}
-stdout_logfile=NONE
-stderr_logfile=NONE
-stdout_events_enabled=true
-stderr_events_enabled=true
+{% if include_exists("supervisor.json") %}
```

Comment:
With cookiecutter there's no way to share template code or macros between different files within the `cookiecutters` repo, so we end up having to duplicate this whole chunk of template code